modules/network/main.tf

```diff
@@ -162,17 +162,17 @@ resource "aws_lb" "km_lb" {
 resource "aws_lb_target_group" "km_lb_target" {
   name        = "km-lb-target-group-${var.environment}"
   port        = 80
-  protocol    = "HTTP"
+  protocol    = "HTTPS"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure one target group is configured to listen on HTTPS for AWS Load balancer through AWS LB Listener**
 
**Vulnerability**
Not configuring AWS Load balancer to have one target group listening on HTTPS ensures end-to-end encryption is not enabled. This could impact the confidentiality of data in transit.
 
**Remediation**
It is recommended to configure one target group listening on HTTPS for AWS Load balancer. This can be achieved by declaring a resource 'aws_lb_target_group' in your iac and make sure 'protocol' field is set to HTTPS.

source: accurics-